src/NosCore.GameObject/ComponentEntities/Interfaces/IAliveEntity.cs

```diff
@@ -23,10 +23,23 @@ public interface IAliveEntity : IVisualEntity
         bool NoAttack { get; set; }
 
         bool NoMove { get; set; }
+
         bool IsAlive { get; set; }
 
         short MapX { get; set; }
 
         short MapY { get; set; }
+
+        int MaxHp { get; }
+
+        int MaxMp { get; }
+
+        byte Level { get; set; }
+
+        long LevelXp { get; set; }
+
+        byte HeroLevel { get; set; }
+
+        long HeroXp { get; set; }
```

Comment:
remove HeroXp and put it inside ICharacter it's only use for character specific things